starknet_devnet/util.py

```diff
@@ -61,6 +80,28 @@ def parse_dump_on(option: str):
         return DumpOn[option.upper()]
     sys.exit(f"Error: Invalid --dump-on option: {option}. Valid options: {DUMP_ON_OPTIONS_STRINGIFIED}")
 
+def int_or_float(value: str):
+    """Parses the value as either an int or a float."""
```

Comment:
But it always return an int, please document that. Perhaps change name to `numberish` or something